.github/workflows/release.yml

```diff
@@ -16,45 +16,73 @@ on:
         description: Build node
         default: true
       runtime:
-        type: boolean
-        description: Build runtimes deterministically
-        default: true
+        description: 'Runtime to build (devnet, testnet, mainnet)'
+        required: true
```

Comment:
If this is required and a choice, does it mean that a runtime will always build when just wanting to build the node, which in turn will build all the runtimes?

Reply:
Good point! See the latest commit and updated PR description.

Reply:
Rather than 'none', would it not be simpler to remove `required: true` and use `github.event.inputs.runtime != ''`?

Reply:
For the choice input it still requires that something is selected even if required is removed. So, I set the default to be "" (empty string), and added a new choice "".